lib/context_adapter_helper.js

```diff
@@ -774,9 +871,8 @@ module.exports = {
   getServiceDescriptor: getServiceDescriptor,
   getOperationDescriptor: getOperationDescriptor,
   sendThirdPartyRequest: sendThirdPartyRequest,
-  notifyOperationInProgress: notifyOperationInProgress,
+  // notifyOperationInProgress: notifyOperationInProgress,
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/b116d5f7db718931e087d47ee388ef3298211c99